Tools/plugins/MuonPogTreeProducer.cc

```diff
@@ -369,6 +369,7 @@ void MuonPogTreeProducer::analyze (const edm::Event & ev, const edm::EventSetup
     {
       nGoodMuons = fillMuons(muons,vertexes,beamSpot);
     }
+  eventId_.nMuons = nGoodMuons;
```

Comment:
Ok sorry, I was understood that you want the max pt difference between all refits, and not the max pt itself. So from my side go ahead, Raffaella could you implemented it? I think it is really a good thing to add. thanks Carlo.